test/data.js

```diff
@@ -5,6 +5,7 @@
 const metatests = require('metatests');
 const common = require('..');
 
+// prettier-ignore
```

Comment:
I don't think this is required here since most of the entries here are already multiline, and it doesn't drastically change readability for the ones that are not.

Reply:
I'm fine with both, wdyt @tshemsedinov?